frontend/app/helpers/spree/navigation_helper.rb

```diff
@@ -57,11 +57,11 @@ def spree_menu(location = 'header')
       method_name = "for_#{location}"
 
       if available_menus.respond_to?(method_name) && Spree::Menu::MENU_LOCATIONS_PARAMETERIZED.include?(location)
-        available_menus.send(method_name, I18n.locale)
+        available_menus.send(method_name, I18n.locale, current_store)
```

Comment:
@MatthewKennedy why do we need to pass `current_store` here? `available_menus` is already scoped to the current store: https://github.com/spree/spree/blob/5ecb98271d927050cc08a0aafda4593a874ce841/core/lib/spree/core/controller_helpers/store.rb#L17

Reply:
I was using that for calling `store.default_menu(location_name)` inside `Spree::Menu`, for the fallback.

Reply:
@damianlegawiec How about the latest commit as an option?